### ISSUE TITLE
[IOS] [App]on open results in WSOD (blank screen)

### ISSUE THREAD

**Issue by petar-nikov** [requestly/customer-support#71]
When opening Requestly on Desktop it results in blank screen with following errors reported in dev tools:
```sh
Uncaught Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Error%3A%20…%3A%20networksetup%20-setwebproxy%20Wi-Fi%20127.0.0.1%208081%3B%0A&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at ji (manageRules.renderer.prod.js:2)
    at manageRules.renderer.prod.js:2
    at Aa (manageRules.renderer.prod.js:2)
    at gs (manageRules.renderer.prod.js:2)
    at ul (manageRules.renderer.prod.js:2)
    at sl (manageRules.renderer.prod.js:2)
    at Js (manageRules.renderer.prod.js:2)
    at manageRules.renderer.prod.js:2
    at t.unstable_runWithPriority (manageRules.renderer.prod.js:2)
    at Wo (manageRules.renderer.prod.js:2)
    at Ko (manageRules.renderer.prod.js:2)
    at Vo (manageRules.renderer.prod.js:2)
    at Ys (manageRules.renderer.prod.js:2)
    at Object.enqueueSetState (manageRules.renderer.prod.js:2)
    at t.w.setState (manageRules.renderer.prod.js:2)
    at t.value (manageRules.renderer.prod.js:2)
    at Object.addNotification (manageRules.renderer.prod.js:2)
    at EventEmitter.<anonymous> (manageRules.renderer.prod.js:2)
    at EventEmitter.emit (events.js:210)
    at Object.onMessage (init.ts:50)
```

**Comment by requestly-team** [comment]
Thank you @petar-nikov for logging this issue. Our team is looking at it and working on the solution. Will keep you posted.

**Comment by petar-nikov** [comment]
@requestly-team Thank you,  if you need more information let me know.

**Comment by requestly-team** [comment]
Hey @petar-nikov. Sorry for the delayed response. We have fixed this issue in v0.0.6-beta. Can you update the app and try again.

**Comment by petar-nikov** [comment]
@requestly-team Thank you it's fixed
